ios/gamma/split-view/RNSSplitViewScreenController.swift

```diff
@@ -18,8 +18,14 @@ public class RNSSplitViewScreenController: UIViewController {
     return splitViewScreenComponentView.reactEventEmitter()
   }
 
+  private var displayLink: CADisplayLink?
+  private var lastAnimationFrame: CGRect?
+  private var transitionInProgress: Bool
```

Comment:
I'd be great for these names to be more informative, as they are used in very specific context: for size transition of the managed view. If you group these in a dedicated struct, then these names are fine.

Reply:
https://github.com/software-mansion/react-native-screens/pull/3176